### ISSUE TITLE
iostream stream error: Access is denied.

### ISSUE THREAD

**Issue by bb30994** [FoldingAtHome/fah-issues#1295]
(Simple enhancement request quoted from https://foldingforum.org/viewtopic.php?p=305171#p305171)
----------------------------------------------------------------------
I have a similar issue using the 'portable version' of FAH.
Since NACL has finished, the binaries found on the site, allow me to run FAH, without installing it, and without admin privileges.
In a way, the same as NaCL, but no browser option.

I have the same issue running fahClient (CPU only) on Windows 10.
And I know it's because the program hasn't started the way it was supposed to start.

But for those who run FAHClient binaries, via a CMD window (with no install),
So far, I get the issue when exiting abruptly (closing CMD window, or repeatedly pressing CTRL + C).
Fah shuts down, but leaves one of the work directories (or files) as read-only (...."fah\work\00" folder)
The issue is resolved by closing FAH, and removing the 'read-only' tag from the '00' folder (right click, properties, remove 'read only' checkbox), and restarting fah.

I'm also running fahClient (CPU) on Windows 7, but haven't seen any access denied errors there.

Also, setting the 'hidden' option in Task Scheduler, to schedule FAH to start at boot (for those without admin rights), the 'hidden' option doesn't work. There's a FAH icon appearing in Windows Systray, but it's non functional, disappears, or causes errors with the cmd line.
Again, due to not running FAH the way it was intended (with installation), but FAH binaries do work without installation of any kind.

So, to summarize, I think the iostream errors can be prevented, by removing the read only lock on the directories.

**Comment by shorttack** [comment]
Closed: Windows enhancement. Not a common problem with users with FAHcontrol.
